content/influxdb/cloud/write-data/troubleshoot.md

```diff
@@ -18,45 +18,48 @@ related:
+For example, a partial write may occur when InfluxDB writes points that conform to the schema of existing data, but rejects points that have the wrong data type in a field.
 
-## HTTP status codes
+## Review HTTP status codes
 
 InfluxDB uses conventional HTTP status codes to indicate the success or failure of a request.
 Write requests return the following status codes:
 
 - `204` **Success**: InfluxDB validated the request data format and accepted the data for writing to the bucket.
     {{% note %}}
-  `204` doesn't indicate a successful write operation since writes are asynchronous.
-  If some of your data did not write to the bucket, see how to [troubleshoot rejected points](#troubleshoot-rejected-points).
+`204` doesn't indicate a successful write operation since writes are asynchronous. If some of your data did not write to the bucket, see how to [check for rejected points](#review-rejected-points).
     {{% /note %}}
 
-- `400` **Bad request**: The [line protocol](/influxdb/cloud/reference/syntax/line-protocol/) data in the request was malformed.
-   The response body contains the first malformed line in the data. All request data was rejected and not written.
+- `400` **Bad request**: The [line protocol](/influxdb/cloud/reference/syntax/line-protocol/) data in the request is malformed.
+   The response body contains the first malformed line in the data. All request data is rejected and not written.
 - `401` **Unauthorized**: May indicate one of the following:
   -  [`Authorization: Token` header](/influxdb/cloud/api-guide/api_intro/#authentication) is missing or malformed.
   - [API token](/influxdb/cloud/api-guide/api_intro/#authentication) value is missing from the header.
-  - API token does not have sufficient permissions to write to the organization and bucket. For more information about token types and permissions, see [Manage API tokens](/influxdb/cloud/security/tokens/)
-- `404` **Not found**: A requested resource (e.g. an organization or bucket) was not found. The response body contains the requested resource type, e.g. "organization", and resource name.
-- `413` **Request entity too large**: The payload exceeded the 50MB limit. All request data was rejected and not written.
+  - API token does not have sufficient permissions to write to the organization and bucket.
+    For more information about token types and permissions, see [Manage API tokens](/influxdb/cloud/security/tokens/)
+- `404` **Not found**: A requested resource (e.g. an organization or bucket) was not found.
+  The response body contains the requested resource type, e.g. "organization", and resource name.
+- `413` **Request entity too large**: The payload exceeded the 50MB limit.
+- `422` **Unprocessible entity**: Request data is invalid. `code` and `message` in the response body provide details about the problem.
```

Comment:
I'm not sure that this error is possible in the write path. Probably worth fixing in another PR though, as it's not related to this one.